processing/src/main/java/org/apache/druid/query/groupby/strategy/GroupByStrategyV2.java

```diff
@@ -450,18 +450,17 @@ public Row apply(final Row row)
                 );
               }
             }, subtotalQuery, null),
-            subtotalQuery
-                             )
+            subtotalQuery)
```

Comment:
This method call is formatted improperly. It should be
```java
results = mergeResults(
    new QueryRunner<Row>()
    {
        @Override
        public Sequence<Row> run(QueryPlus<Row> queryPlus, ResponseContext responseContext)
        {
          return GroupByRowProcessor.getRowsFromGrouper(query, null, grouperSupplier);
        }
    },
    subtotalQuery,
    null
);
subtotalsResults.add(applyPostProcessing(results, subtotalQuery));
```

Reply:
Good catch, thanks